internal/datamodel/apply_computed_project_quota.go

```diff
@@ -374,7 +375,14 @@ func acpqComputeQuotas(stats map[limes.AvailabilityZone]clusterAZAllocationStats
 				}
 			}
 			if sumOfLocalizedQuotas < cfg.ProjectBaseQuota {
-				target[limes.AvailabilityZoneAny][resourceID].Desired = cfg.ProjectBaseQuota - sumOfLocalizedQuotas
+				// AZ separated toplogy receives the basequota to all available AZs
+				if resInfo.Topology == liquid.AZSeparatedResourceTopology {
+					for az := range isRelevantAZ {
+						target[az][resourceID].Desired = cfg.ProjectBaseQuota - target[az][resourceID].Allocated
```

Comment:
`Desired` is not computed on top of `Allocated`:
```suggestion
						target[az][resourceID].Desired = cfg.ProjectBaseQuota
```